_data/packages.yml

```diff
@@ -652,7 +747,7 @@
   repository_link: https://github.com/unionai-oss/pandera
   version_submitted: 0.1.5
   categories:
-    - data-munging
+    - data munging
```

Comment:
This might cause an issue with how i'm using jekyll to populate categories on the packages page. i think we need that dash here for it to work properly as that becomes an id tag i believe